pkg/process/util/orchestrator/pod.go

```diff
@@ -186,9 +186,53 @@ func extractPodMessage(p *v1.Pod) *model.Pod {
 	podModel.Status = ComputeStatus(p)
 	podModel.ConditionMessage = GetConditionMessage(p)
 
+	for _, c := range p.Spec.Containers {
+		modelReq := convertResourceRequirements(c.Resources, c.Name, model.ResourceRequirementsType_container)
+		podModel.ResourceRequirements = append(podModel.ResourceRequirements, modelReq)
+	}
+
+	for _, c := range p.Spec.InitContainers {
+		modelReq := convertResourceRequirements(c.Resources, c.Name, model.ResourceRequirementsType_initContainer)
+		podModel.ResourceRequirements = append(podModel.ResourceRequirements, modelReq)
+	}
+
 	return &podModel
 }
 
+func convertResourceRequirements(rq v1.ResourceRequirements, containerName string, resourceType model.ResourceRequirementsType) *model.ResourceRequirements {
+	requests := map[string]int64{}
+	limits := map[string]int64{}
+
+	cpuLimit := rq.Limits.Cpu()
+	if !cpuLimit.IsZero() {
+		limits[v1.ResourceCPU.String()] = cpuLimit.Value()
+	}
+
+	memLimit := rq.Limits.Memory()
+	if !memLimit.IsZero() {
+		limits[v1.ResourceMemory.String()] = memLimit.Value()
+	}
+
+	cpuRequest := rq.Requests.Cpu()
+	if !cpuRequest.IsZero() {
+		requests[v1.ResourceCPU.String()] = cpuRequest.Value()
+	}
+
+	memRequest := rq.Requests.Memory()
+	if !memRequest.IsZero() {
+		// TODO: check is `value` correct?
+		// TODO: check whether we want to save "empty" values and send them as well. Kind of empty sliecs only filed with the containerName and type.
```

Comment:
doublecheck: do we want to send "empty slices, only filled with types and names" to our backend?